packages/ag-charts-community/src/util/json.ts

```diff
@@ -246,6 +246,7 @@ export type JsonApplyParams = {
  *
  * @param target to apply source JSON properties into
  * @param source to be applied
+ * @param params
```

Comment:
Needed?

Reply:
Not really. My IDE JSDoc analyzers just yells at me that it isn't defined as part of the JSDoc 😅